docs/getting-started/build-a-sample-app.md

```diff
@@ -23,7 +44,14 @@ edition = "2018"
 yew = "0.17"
 ```
 
-Copy the following template into your `src/main.rs` file:
+### Update main.rs
+
+Replace the contents of `src/main.rs` with the following code.
+
+This template sets up a root `Component` called `Model` which renders a button that updates itself when clicked.
+Take special note of `yew::start_app::<Model>()` inside `main()` which starts your app and mounts it to the page's `<body>` tag.
+If you would like to start your application with any dynamic properties, you can instead use `yew::start_app_with_props::<Model>(..)`.
+
```

Comment:
I think the lines explaining the snippet should either go below (like it was before) or be in a more conversational tone. Example:

> We need to a template which sets up a root `Component` called `Model` which renders a button that updates itself when clicked.
> ...
> Now, let's replace the contents of `src/main.rs` with the following code.

Feel free to change the wording here

Reply:
Complete